src/browser/ui/dom/DOMPropertyOperations.js

```diff
@@ -111,6 +126,22 @@ var DOMPropertyOperations = {
   },
 
   /**
+   * Creates markup for a custom property.
+   *
+   * @param {string} tagName
+   * @param {string} name
+   * @param {*} value
+   * @return {?string} Markup string, or null if the property was invalid.
+   */
+  createMarkupForCustomAttribute: function(name, value) {
```

Comment:
That's how I had my original code.  Sebastian chewed me out for it :), and said that we shouldn't be leaking tagName into this file (see first comment on this diff in the history).

For now, I'm tempted to leave it as-is, since this is how Sebastian wanted it.  You guys can battle it out later.